src/main/java/athena/TimeSlot.java

```diff
@@ -0,0 +1,42 @@
+package athena;
+
+import athena.exceptions.NoNextSlotException;
+
+public class TimeSlot {
+    private Log dayLog;
+    private final Integer wake = 8;
+    private final Integer sleep = 22;
```

Comment:
Hmmm why is sleep 22 here

Reply:
It's a magic number, need to sync up with the timetable. It's to tell what is a valid time slot. If it goes over this number then it stops

Reply:
Our sleep time is 12am so should it be 24 instead of 22?

Reply:
Yup, it was that originally. I think I changed it because of the demo